packaging/ios/build.sh

```diff
@@ -241,4 +241,4 @@ cd build-Subsurface-mobile-Qt_$(echo ${QT_VERSION} | tr . _)_for_iOS-${DEBUGRELE
 ${IOS_QT}/${QT_VERSION}/ios/bin/qmake ../Subsurface-mobile/Subsurface-mobile.pro \
 	-spec macx-ios-clang CONFIG+=$TARGET CONFIG+=$TARGET2 CONFIG+=$DRCONFIG
 
-make qmake_all
+make all
```

Comment:
This may have changed from when I first did this. The rationale was that this would get things set up so that I could then use Xcode to upload the app. Otherwise everything is built twice...